welcome/welcome.py

```diff
@@ -289,6 +289,7 @@ def __init__(self, *args, **kwargs) -> None:
         Allows for the following customizations:
           `{member}` is the member who joined
           `{server}` is the server
+          `{roles}` is a list of all the roles that the member has at the time
```

Comment:
I don't think this works here. For DMs the message is built and sent by `__dm_user()`, which doesn't format the roles in. It could, but I don't think it should. The user who just joined probably doesn't need to be told what roles they have right now.

Reply:
makes sense, I will remove it.